docs/source/howto/index.rst

```diff
@@ -5,7 +5,9 @@ How-To Guides
 .. toctree::
    :maxdepth: 1
 
-   codes
+   run_codes
+   ssh
```

Comment:
I think these can and probably should be merged

Reply:
let's discuss this at the top PR level